javascript/app-event-analysis/src/components/Legend.tsx

```diff
@@ -36,7 +36,7 @@ export const Legend: React.FC<LegendProps> = memo(function Legend({
 				.attr('width', legendItemSize)
 				.attr('height', legendItemSize)
 				.style('fill', d => d.color)
-				.attr('opacity', BAR_TRANSPARENT)
+				.attr('opacity', d => d.opacity || BAR_TRANSPARENT)
```

Comment:
should we use ?? instead of || ? 

Is an opacity of zero valid?

Reply:
No, I don't think so